Meetings/devel-call-2025-04-09.md

```diff
@@ -0,0 +1,38 @@
+# Meeting Minutes: SVSM Development Call (April 9th, 2025)
+
```

Comment:
The list of participants is missing, I don't know if it is intended or not.

Reply:
It is intended. GMeet doesn't capture this anymore in the notes or transcript and I am not tracking it myself.

Reply:
ah, I see, yeah let's skip it in that case.